assets/script/app.js

```diff
@@ -340,4 +340,65 @@ function checkForUndefined(restaurantInfo){
     return "Not available"
   }
   else return restaurantInfo
+}
+
+/* Local Storage */
+
+const recentSearchesSize = 5;
+const recentSearchesKey = 'recentSearches';
+
+const searchButtonElement = $("#submitButton");
+const searchValueElement = $("#userLocationInput");
+const searchResultsListElement = $("search-results-list");
+const recentSearchListElement = $('#recent-search-list');
+const recentSearchClearButtonElement = $('#recent-search-clear');
+
+searchButtonElement.click(searchRestaurants);
+recentSearchClearButtonElement.click(clearRecentSearches);
+
+renderRecentSearches();
+
+function searchRestaurants(event){
+  event.preventDefault();
+  const searchValue = searchValueElement.val();
+  addRecentSearches(searchValue)
+  renderRecentSearches();
+  searchResultsListElement.html("");
```

Comment:
lines 366 to 370 seems to be doing the same thing as the renderRecentSearches function